Mathlib/MeasureTheory/Constructions/Convolution.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2023 Josha Dekker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Josha Dekker
+-/
+import Mathlib.MeasureTheory.Constructions.Prod.Basic
+import Mathlib.MeasureTheory.Measure.MeasureSpace
+/-!
+# The multiplicative and additive convolution of measures
+
+In this file we define and prove properties about the convolutions of two measures
+
+## Main definitions
+
+* `MeasureTheory.Measure.mconv`: The multiplicative convolution of two measures.
+* `MeasureTheory.Measure.conv`: The additive convolution of two measures.
+
+## Main results
+
+* add
+## Implementation Notes
+
+## Tags
+
```

Comment:
It looks like this file docstring is still WIP?

Reply:
That one can probably go, I wouldn't say there is a big "main result" at this point.